airflow/config_templates/config.yml

```
@@ -233,6 +233,14 @@
       type: string
       example: ~
       default: "True"
+    - name: dag_ignore_file_syntax
+      description: |
+        The pattern syntax used in the ".airflowignore" files in the DAG directories. Valid values are
+        ``regexp`` or ``glob``.
```

```suggestion
        The pattern syntax used in the ".airflowignore" files in the DAG directories. Valid values are
        ``regexp`` and ``glob``.
```

I think this is correct English? (I’m non-native so don’t take my words)

---

I think either can work actually in this context, I chose "or" since you have to choose one or the other.

---

Either makes sense for me too